lib/options.js

```diff
@@ -17,35 +17,49 @@ const discordAirtableBadgeMap = assertToken(
 	x => x,
 	x => JSON.parse(x)
 )
-const discordBadgeChannelId = assertToken(`DISCORD_BADGE_CHANNEL_ID`)
+const discordBadgeChannelIdNwif = assertToken(`DISCORD_BADGE_CHANNEL_ID_NWIF`)
+const discordBadgeChannelIdScif = assertToken(`DISCORD_BADGE_CHANNEL_ID_SCIF`)
 
 
 const Discord = {
 	botToken: discordBotToken,
 	attendeeGuildID: discordAttendeeGuildID,
 	staffGuildID: discordStaffGuildID,
 	airtableBadgeMap: discordAirtableBadgeMap,
-	badgeChannelId: discordBadgeChannelId,
+	badgeChannelIdNwif: discordBadgeChannelIdNwif,
+	badgeChannelIdScif: discordBadgeChannelIdScif
 }
 
-const airtableAPIToken = assertToken(`AIRTABLE_API_TOKEN`)
-const airtableBaseID = assertToken(`AIRTABLE_BASE_ID`)
-const Airtable = {
-	apiToken: airtableAPIToken,
-	baseID: airtableBaseID,
+const airtableAPITokenNwif = assertToken(`AIRTABLE_API_TOKEN_NWIF`)
+const airtableAPITokenScif = assertToken(`AIRTABLE_API_TOKEN_SCIF`)
+const airtableBaseIDNwif = assertToken(`AIRTABLE_BASE_ID_NWIF`)
+const airtableBaseIDScif = assertToken(`AIRTABLE_BASE_ID_SCIF`)
+const AirtableNwif = {
+	apiToken: airtableAPITokenNwif,
+	baseID: airtableBaseIDNwif,
+}
+const AirtableScif = {
+	apiToken: airtableAPITokenScif,
+	baseID: airtableBaseIDScif,
 }
 
 const scheduleUrl = assertToken(`SCHEDULE_URL`)
+const nodeEnv = assertToken(`NODE_ENV`)
```

Comment:
Doesn't look like this gets used

Reply:
Imported and used in badge-notifier to show a "this is a test" warning: https://github.com/IdolFest/idolfest-discord/pull/4/files#diff-fd7913345d10ec1d693fbe5fd23dce5e935dc25e740d85c0a1ae78242c4ff25bR52